app/scripts/metamask-controller.js

```diff
@@ -3218,7 +3218,6 @@ export default class MetamaskController extends EventEmitter {
         accountsController.setAccountName.bind(accountsController),
 
       setAccountLabel: (address, label) => {
-        this.preferencesController.setAccountLabel(address, label);
```

Comment:
No longer needed. The preferences controller mirrors the accounts controller state, `setAccountLabel` just delegates the call back to the accounts controller.